app/components/task-list/multiple-sections.njk

```diff
@@ -0,0 +1,118 @@
+
+  <div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper" id="maincontent">
+
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+          <h1 class="nhsuk-heading-l">Service name</h1>
+
+          <h2 class="nhsuk-heading-m">Check before you start</h2>
+          {{ taskList({
+            idPrefix: "before-you-start",
+            items: [
+              {
+                title: {
+                  text: "Check eligibility"
+                },
+                href: "#",
+                status: {
+                  text: "Completed",
+                  classes: "nhsuk-task-list__status--completed"
+                }
+              },
+              {
+                title: {
+                  text: "Read declaration"
+                },
+                href: "#",
+                status: {
+                  tag: {
+                    text: "Incomplete",
+                    classes: "nhsuk-tag--blue"
+                  }
+                }
+              }
+            ]
+          }) }}
+
+          <h2 class="nhsuk-heading-m nhsuk-u-margin-top-3">Prepare application</h2>
+
+          {{ taskList({
+            idPrefix: "your-health",
+            items: [
+              {
+                title: {
+                  text: "Exercise"
+                },
+                href: "#",
+                status: {
+                  text: "Completed",
+                  classes: "nhsuk-task-list__status--completed"
+                }
+              },
+              {
+                title: {
+                  text: "Personal health"
+                },
+                href: "#",
+                status: {
+                  text: "Completed",
+                  classes: "nhsuk-task-list__status--completed"
+                }
+              },
+              {
+                title: {
+                  text: "Family health history"
+                },
+                hint: {
+                  text: "Details of your parents and siblings"
+                },
+                href: "#",
+                status: {
+                  tag: {
+                    text: "Incomplete",
+                    classes: "nhsuk-tag--blue"
+                  }
+                }
+              },
+              {
+                title: {
+                  text: "Smoking history"
+                },
+                href: "#",
+                status: {
+                  tag: {
+                    text: "Incomplete",
+                    classes: "nhsuk-tag--blue"
+                  }
+                }
+              },
+              {
+                title: {
+                  text: "Blood test"
+                },
+                status: {
+                  text: "Cannot start yet",
+                  classes: "nhsuk-task-list__status--cannot-start-yet"
+                }
+              }
+            ]
+          }) }}
+
+          {{ button({
+            "text": "Continue"
```

Comment:
Nitpick.

```suggestion
            text: "Continue"
```

Reply:
This PR probably predates these changing